src/Microsoft.TemplateEngine.Utils/InMemoryFileSystem.cs

```diff
@@ -848,5 +848,40 @@ public DateTime GetLastWriteTimeUtc(string file)
 
             return targetFile.LastWriteTimeUtc;
         }
+
+        public void SetLastWriteTimeUtc(string file, DateTime lastWriteTimeUtc)
+        {
+            if (!IsPathInCone(file, out string processedPath))
+            {
+                if (_basis is IFileLastWriteTimeSource lastWriteTimeSource)
```

Comment:
minor:  in other parts, one liner "if" statements ' bodies are enclosed in { }.  For homogeneity, should this if statement do the same?

Reply:
Essentially the same as the others - this was copied from `GetLastWriteTimeUtc`.  I'll fix it up for both.